scripts/MantidIPython/plot_functions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Plotting functions for use in IPython notebooks that are generated by MantidPlot
+"""
+
+import matplotlib.pyplot as plt
+import sys
+import os
+
+# Import Mantid
+CURRENT_DIR = os.getcwd()
```

Comment:
Why is it necessary to do this? I thought the `mantidpython` startup script set the environment so that mantid could be imported?